Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +15,141 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
+            string[] terms = equation.Split(new Char[] { '*', '=' });
+            /*The first operand has a missing digit*/
+            if (terms[0].Contains('?'))

```

Comment:
Can you please make this code simplified by moving codes in function?
You can read off duplicate code with this.

Reply:
Sir, I updated and made functions with meaningful names for the same. Please check. @arpit-nagar